index.js

```diff
@@ -1,3 +1,8 @@
+var clickCounter = 0
```

Comment:
when declaring multiple variables you can comma chain them like this:
`var a = 1, b = 2, c = 3`

Reply:
Group style guide - group similar vars on one line - eg.
clickedCard1, clickedCard2